README.md

```diff
@@ -1,295 +1,202 @@
+would also create the first and next folder.
 
 ### Use `mv` to Move or Rename Files and Directories
 
-Move, or `mv` is a command that moves one or more files or directories from one place
-to another.  To move a file from the current directory to another location, enter a
-path as the third word on the command line.
+Move, or `mv` is a command that moves one or more files or directories from one
+place to another.  Alternatively, think of it as a copy, followed by a delete of
+the original. To move a file from the current directory to another location,
+enter a path as the third word on the command line.
 
 ```bash
 $ mv filename /dir1/
 ```
 
-We can also rename file or directory using the `mv` command. To rename a file with
-`mv`, the third word on the command line must end in the new filename.
+We can also rename file or directory using the `mv` command. To rename a file
+with `mv`, the third word on the command line must end in the new filename.
 
 ```bash
 $ mv original_program.rb renamed_program.rb
 ```
 
-### Create Empty Files With `touch`
+### Use `cp` to Copy Files and Directories
 
-We can use the `touch` command to create empty files in the current directory. Try:
+The `cp` command stands for "copy" and copies files and folders, just as
+expected. This command is used to copy files or group of files or directory. It
+creates an exact image of a file on a disk with different file name. This
+command require at least two filenames in its arguments. The flag -r recursively
+copies subfolders and files.
 
-```bash
-$ touch hello_world.rb
-```
-
-Now try:
+Try copying the file we just created:
 
 ```bash
-$ ls
+$ cp hello_world.rb hello_world_again.rb
 ```
 
-You should see the file you just created, `hello_world.rb`, in the working directory.
-Note that this is an empty file and has nothing inside of it, because you just created it.
+* If the command contains *two* file names, then it copy the contents of 1st
+  file to the 2nd file. If the 2nd file doesn’t exist, then first it creates one
+  and content is copied to it. But if it existed then it is simply overwritten
+  without any warning. So be careful when you choose destination file name.
 
-### Making New Directiries Using `mkdir`
-
-We can make directories with the `mkdir` command:
+* If the command has one or more arguments, specifying file names and following
+  those arguments, an argument specifying directory name then this command
+  copies each source file to the destination directory with the same name. It
+  will create the directory if it doesn't exist, but **if it exists then it will
+  be overwritten**, so be careful!
 
 ```bash
-$ mkdir name_of_directory
+$ cp hello_world.rb hello_world_again.rb name_of_directory
 ```
-Now if you enter `ls` you should see the empty directory you just created in the working
-directory.
+Now you have too many files, or some of them are no longer needed. Let's talk
+about how to remove files.
 
 ### Removing Files With `rm`
 
```

Comment:
Let's put a nice big warning up here..like..hey we've told you not to be afraid, but here's hte part where you can really break stuff. Deleting files is a natural part of life, but if you're in the wrong spot you could do some real damage here.